hermes-console-vue/json-server/routes.json

```diff
@@ -3,10 +3,13 @@
   "/workload-constraints": "/constraints",
   "/consistency/inconsistencies/topics": "/inconsistentTopics",
   "/topics/:id/metrics": "/topicsMetrics/:id",
+  "/topics/:id/preview": "/topicPreview",
+  "/topics/:id/subscriptions": "/topicSubscriptions",
   "/topics/:topicName/subscriptions/:id": "/subscriptions/:id",
   "/topics/:topicName/subscriptions/:id/consumer-groups": "/consumerGroups",
   "/topics/:topicName/subscriptions/:id/health": "/subscriptionsHealth/:id",
   "/topics/:topicName/subscriptions/:id/metrics": "/subscriptionsMetrics/:id",
   "/topics/:topicName/subscriptions/:id/undelivered": "/subscriptionUndeliveredMessages",
-  "/topics/:topicName/subscriptions/:id/undelivered/last": "/subscriptionUndeliveredMessages/:id"
-}
\ No newline at end of file
+  "/topics/:topicName/subscriptions/:id/undelivered/last": "/subscriptionUndeliveredMessages/:id",
+  "/owners/sources/Service%20Catalog/:id": "/topicsOwners/:id"
```

Comment:
I avoid putting TODOs in code, especially in early state of project, bc there would be too much places to mark :P This will be adjusted in next task, as said above